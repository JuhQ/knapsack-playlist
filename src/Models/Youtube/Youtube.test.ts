import { YoutubeMusic, YoutubeAudiobooks } from "./Youtube";

describe("YouTube model", () => {
  it("should return list of audiobooks", () => {
    expect(YoutubeAudiobooks()).toMatchSnapshot();
  });

  it("should return list of music", () => {
    expect(YoutubeMusic()).toMatchSnapshot();
  });
});
