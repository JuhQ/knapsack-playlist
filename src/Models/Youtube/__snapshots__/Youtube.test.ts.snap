// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YouTube model should return list of music 1`] = `
Array [
  Object {
    "id": "gAjR4_CbPpQ",
    "seconds": 223,
    "title": "Daft Punk - Harder Better Faster (Official Video)",
  },
  Object {
    "id": "HhZaHf8RP6g",
    "seconds": 407,
    "title": "Daft Punk - Veridis Quo",
  },
  Object {
    "id": "sOS9aOIXPEk",
    "seconds": 229,
    "title": "Daft Punk - Something About Us (Official Video)",
  },
  Object {
    "id": "3MteSlpxCpo",
    "seconds": 296,
    "title": "[Official Video] Daft Punk - Pentatonix",
  },
  Object {
    "id": "FxzBvqY5PP0",
    "seconds": 265,
    "title": "Daft Punk - Digital Love",
  },
  Object {
    "id": "1V_xRb0x9aw",
    "seconds": 269,
    "title": "Gorillaz - Clint Eastwood (Official Video)",
  },
  Object {
    "id": "djV11Xbc914",
    "seconds": 244,
    "title": "a-ha - Take On Me (Official 4K Music Video)",
  },
  Object {
    "id": "ub747pprmJ8",
    "seconds": 229,
    "title": "Right Here, Right Now by Fatboy Slim [Official Video]",
  },
  Object {
    "id": "wCDIYvFmgW8",
    "seconds": 232,
    "title": "Fatboy Slim - Weapon Of Choice [Official Video]",
  },
  Object {
    "id": "D8K90hX4PrE",
    "seconds": 176,
    "title": "Daft Punk - Technologic",
  },
  Object {
    "id": "UclCCFNG9q4",
    "seconds": 275,
    "title": "Gorillaz - Clint Eastwood",
  },
  Object {
    "id": "sFZjqVnWBhc",
    "seconds": 197,
    "title": "Daft Punk - Robot Rock (Official Video)",
  },
  Object {
    "id": "bRt5z880CFY",
    "seconds": 235,
    "title": "Daft Punk - Superheroes",
  },
  Object {
    "id": "HyHNuVaZJ-k",
    "seconds": 254,
    "title": "Gorillaz - Feel Good Inc. (Official Video)",
  },
  Object {
    "id": "fPO76Jlnz6c",
    "seconds": 257,
    "title": "Coolio - Gangsta's Paradise (feat. L.V.) [Official Music Video]",
  },
  Object {
    "id": "FQlAEiCb8m0",
    "seconds": 264,
    "title": "Stardust - Music Sounds Better With You (Official Music Video)",
  },
  Object {
    "id": "L93-7vRfxNs",
    "seconds": 213,
    "title": "Daft Punk - Aerodynamic",
  },
  Object {
    "id": "KQ6zr6kCPj8",
    "seconds": 376,
    "title": "LMFAO ft. Lauren Bennett, GoonRock - Party Rock Anthem (Official Video)",
  },
  Object {
    "id": "mmi60Bd4jSs",
    "seconds": 335,
    "title": "Daft Punk - Da Funk",
  },
  Object {
    "id": "mMfxI3r_LyA",
    "seconds": 222,
    "title": "Modjo - Lady (Hear Me Tonight) (Official Video)",
  },
  Object {
    "id": "04mfKJWDSzI",
    "seconds": 267,
    "title": "Gorillaz - On Melancholy Hill (Official Video)",
  },
  Object {
    "id": "8UVNT4wvIGY",
    "seconds": 244,
    "title": "Gotye - Somebody That I Used To Know (feat. Kimbra) - official music video",
  },
  Object {
    "id": "HCfPhZQz2CE",
    "seconds": 600,
    "title": "Daft Punk - 'Get Lucky' (10 min loop)",
  },
  Object {
    "id": "Hy8kmNEo1i8",
    "seconds": 211,
    "title": "Scatman (ski-ba-bop-ba-dop-bop) Official Video HD -Scatman John",
  },
  Object {
    "id": "MV_3Dpw-BRY",
    "seconds": 257,
    "title": "Kavinsky - Nightcall (Drive Original Movie Soundtrack) (Official Audio)",
  },
  Object {
    "id": "dKJfJMMsqX4",
    "seconds": 239,
    "title": "Daft Punk - Face To Face",
  },
  Object {
    "id": "nhPaWIeULKk",
    "seconds": 303,
    "title": "Gorillaz - Stylo (Official Video)",
  },
  Object {
    "id": "cLnkQAeMbIM",
    "seconds": 301,
    "title": "Gorillaz - Dirty Harry (Official Video)",
  },
  Object {
    "id": "a5uQMwRMHcs",
    "seconds": 340,
    "title": "Daft Punk ft. Julian Casablancas - Instant Crush (Official Video)",
  },
  Object {
    "id": "NF-kLy44Hls",
    "seconds": 251,
    "title": "Daft Punk - Lose Yourself to Dance (Official Version)",
  },
  Object {
    "id": "PwILkY9gRrc",
    "seconds": 329,
    "title": "Daft Punk - Da Funk (Official audio)",
  },
  Object {
    "id": "uAOR6ib95kQ",
    "seconds": 288,
    "title": "Gorillaz - DARE (Official Video)",
  },
  Object {
    "id": "pMyG_H6sHlM",
    "seconds": 370,
    "title": "Get lucky",
  },
  Object {
    "id": "VFZNvj-HfBU",
    "seconds": 246,
    "title": "Daði Freyr (Daði & Gagnamagnið) – Think About Things (Official Video)",
  },
  Object {
    "id": "QK8mJJJvaes",
    "seconds": 233,
    "title": "MACKLEMORE & RYAN LEWIS - THRIFT SHOP FEAT. WANZ (OFFICIAL VIDEO)",
  },
  Object {
    "id": "iU7oF4OXZSE",
    "seconds": 354,
    "title": "Lose Yourself to Dance",
  },
  Object {
    "id": "pXRviuL6vMY",
    "seconds": 226,
    "title": "twenty one pilots: Stressed Out [OFFICIAL VIDEO]",
  },
  Object {
    "id": "4JkIs37a2JE",
    "seconds": 237,
    "title": "Jamiroquai - Virtual Insanity (Official Video)",
  },
  Object {
    "id": "EDwb9jOVRtU",
    "seconds": 327,
    "title": "Madonna - Hung Up (Official Music Video)",
  },
  Object {
    "id": "XgztfRBc2jM",
    "seconds": 229,
    "title": "Eiffel 65 - Blue [Official Video]",
  },
  Object {
    "id": "5qJp6xlKEug",
    "seconds": 374,
    "title": "Gorillaz - Saturnz Barz (Spirit House)",
  },
  Object {
    "id": "dQw4w9WgXcQ",
    "seconds": 213,
    "title": "Rick Astley - Never Gonna Give You Up (Video)",
  },
  Object {
    "id": "FAD_5NWzM5g",
    "seconds": 242,
    "title": "Estelle - American Boy (feat. Kanye West) [Official Video]",
  },
  Object {
    "id": "qmsbP13xu6k",
    "seconds": 187,
    "title": "Mr Oizo - Flat beat (Official Video with Flat Eric - 1999 - F Communications)",
  },
  Object {
    "id": "YlUKcNNmywk",
    "seconds": 322,
    "title": "Red Hot Chili Peppers - Californication [Official Music Video]",
  },
  Object {
    "id": "h61QG4s0I3U",
    "seconds": 238,
    "title": "The Supermen Lovers (feat. Mani Hoffman) · Starlight (Official Video)",
  },
  Object {
    "id": "9Dz6hK585HM",
    "seconds": 594,
    "title": "I Remember",
  },
  Object {
    "id": "NI6aOFI7hms",
    "seconds": 203,
    "title": "The Cardigans - Lovefool",
  },
  Object {
    "id": "zK1mLIeXwsQ",
    "seconds": 594,
    "title": "deadmau5 & Kaskade - I Remember (HQ)",
  },
  Object {
    "id": "CJinWua98NA",
    "seconds": 283,
    "title": "Capital Cities - Kangaroo Court (Official Music Video)",
  },
  Object {
    "id": "QtXby3twMmI",
    "seconds": 316,
    "title": "Coldplay - Adventure Of A Lifetime (Official Video)",
  },
  Object {
    "id": "em0MknB6wFo",
    "seconds": 233,
    "title": "Daft Punk - Something About Us (Official audio)",
  },
  Object {
    "id": "JtYa0rb_jGQ",
    "seconds": 250,
    "title": "Night Club - \\"Show It 2 Me\\" (Official 360 Video)",
  },
  Object {
    "id": "v0NSeysrDYw",
    "seconds": 219,
    "title": "Bob Sinclar - Love Generation",
  },
  Object {
    "id": "y6120QOlsfU",
    "seconds": 233,
    "title": "Darude - Sandstorm",
  },
  Object {
    "id": "pT68FS3YbQ4",
    "seconds": 192,
    "title": "The Strokes - You Only Live Once (Official Music Video)",
  },
  Object {
    "id": "A2VpR8HahKc",
    "seconds": 321,
    "title": "Daft Punk - One more time (Official audio)",
  },
  Object {
    "id": "5NV6Rdv1a3I",
    "seconds": 249,
    "title": "Daft Punk - Get Lucky (Official Audio) ft. Pharrell Williams, Nile Rodgers",
  },
  Object {
    "id": "2zNSgSzhBfM",
    "seconds": 424,
    "title": "MACKLEMORE & RYAN LEWIS - CAN'T HOLD US FEAT. RAY DALTON (OFFICIAL MUSIC VIDEO)",
  },
  Object {
    "id": "qO_IkNfT3Iw",
    "seconds": 13195,
    "title": "History of the Indian wars",
  },
  Object {
    "id": "LL-gyhZVvx0",
    "seconds": 261,
    "title": "Daft Punk ft. Panda Bear - Doin' it Right (Official Audio)",
  },
  Object {
    "id": "qTsaS1Tm-Ic",
    "seconds": 236,
    "title": "Basshunter DotA (Offical Video)",
  },
  Object {
    "id": "Q0oIoR9mLwc",
    "seconds": 304,
    "title": "Red Hot Chili Peppers - Dark Necessities [OFFICIAL VIDEO]",
  },
  Object {
    "id": "PWgvGjAhvIw",
    "seconds": 303,
    "title": "OutKast - Hey Ya! (Official Music Video)",
  },
  Object {
    "id": "uGMqAcLFtuI",
    "seconds": 370,
    "title": "Get Lucky",
  },
  Object {
    "id": "9W44NWYwa1g",
    "seconds": 198,
    "title": "Gorillaz - Andromeda (Official Audio)",
  },
  Object {
    "id": "qFLhGq0060w",
    "seconds": 298,
    "title": "The Weeknd - I Feel It Coming ft. Daft Punk (Official Video)",
  },
  Object {
    "id": "6NXnxTNIWkc",
    "seconds": 299,
    "title": "4 Non Blondes - What's Up (Official Video)",
  },
  Object {
    "id": "h_D3VFfhvs4",
    "seconds": 566,
    "title": "Michael Jackson - Smooth Criminal (Official Video)",
  },
  Object {
    "id": "HpZvM1BaV3U",
    "seconds": 238,
    "title": "Colonel Bagshot - Six Day War - YouTube 360 video",
  },
  Object {
    "id": "-e_3Cg9GZFU",
    "seconds": 371,
    "title": "Skrillex - Summit (feat. Ellie Goulding) [Video by Pilerats]",
  },
  Object {
    "id": "yURRmWtbTbo",
    "seconds": 254,
    "title": "Michael Jackson - Don’t Stop 'Til You Get Enough (Official Video)",
  },
  Object {
    "id": "iTFrCbQGyvM",
    "seconds": 585,
    "title": "Underworld - Born Slippy",
  },
  Object {
    "id": "4D7u5KF7SP8",
    "seconds": 370,
    "title": "Get Lucky",
  },
  Object {
    "id": "lVaBvyzuypw",
    "seconds": 358,
    "title": "Gorillaz - Saturnz Barz (Spirit House) 360",
  },
  Object {
    "id": "iArXv64tCJA",
    "seconds": 278,
    "title": "Peter Bjorn And John - Young Folks",
  },
  Object {
    "id": "19WUwZYM7bM",
    "seconds": 286,
    "title": "Ida Corr vs Fedde Le Grand Let Me Think About It (Extended)",
  },
  Object {
    "id": "Gs069dndIYk",
    "seconds": 216,
    "title": "Earth, Wind & Fire - September",
  },
  Object {
    "id": "QSwilU0kJF0",
    "seconds": 371,
    "title": "David Guetta - The World Is Mine",
  },
  Object {
    "id": "GAly0cOLMc8",
    "seconds": 620,
    "title": "The Weeknd - Blinding Lights (Extended 10 Minute Loop)",
  },
  Object {
    "id": "MaO-PvokDMw",
    "seconds": 473,
    "title": "Don't Stop the Music",
  },
  Object {
    "id": "qL7zrWcv6XY",
    "seconds": 244,
    "title": "Drake - One Dance",
  },
  Object {
    "id": "fHI8X4OXluQ",
    "seconds": 204,
    "title": "The Weeknd - Blinding Lights (Official Audio)",
  },
  Object {
    "id": "hAx6mYeC6pY",
    "seconds": 244,
    "title": "Sophie Ellis-Bextor - Murder On The Dancefloor",
  },
  Object {
    "id": "N0f-QkEVU7U",
    "seconds": 123,
    "title": "6 NASA Technologies to Get Humans to Mars",
  },
  Object {
    "id": "I7HahVwYpwo",
    "seconds": 290,
    "title": "The Black Eyed Peas - Meet Me Halfway (Official Music Video)",
  },
  Object {
    "id": "Rr8ljRgcJNM",
    "seconds": 259,
    "title": "Rammstein - Amerika (Official Video)",
  },
  Object {
    "id": "1M4ADcMn3dA",
    "seconds": 360,
    "title": "Rammstein - Keine Lust (Official Video)",
  },
  Object {
    "id": "8AHCfZTRGiI",
    "seconds": 229,
    "title": "Johnny Cash - Hurt (Official Music Video)",
  },
  Object {
    "id": "hC869VWk9_A",
    "seconds": 259,
    "title": "Rammstein - Reise, Reise",
  },
  Object {
    "id": "YgGzAKP_HuM",
    "seconds": 239,
    "title": "RASPUTIN - Vladimir Putin - Love The Way You Move (Funk Overload) @slocband",
  },
  Object {
    "id": "l3QNCrl6EqQ",
    "seconds": 365,
    "title": "Rammstein - Deutschland (FULL COVER)",
  },
  Object {
    "id": "oVWEb-At8yc",
    "seconds": 370,
    "title": "SABATON - Bismarck (Official Music Video)",
  },
  Object {
    "id": "af59U2BRRAU",
    "seconds": 240,
    "title": "Rammstein - Rosenrot (Official Video)",
  },
  Object {
    "id": "kFr8SdOS5ic",
    "seconds": 281,
    "title": "The Prodigy - The Day Is My Enemy (Fury Road)",
  },
  Object {
    "id": "lV6x2Ctk6CI",
    "seconds": 327,
    "title": "Rammstein - Du hast (Live in Russia Trailer, Multicam By Vinz)",
  },
  Object {
    "id": "oGSLGtTFviA",
    "seconds": 273,
    "title": "Rammstein - Sonne (Official Audio)",
  },
  Object {
    "id": "GX0GXhDLaC4",
    "seconds": 268,
    "title": "Top 10 Craziest Rammstein Moments",
  },
  Object {
    "id": "v4xZUr0BEfE",
    "seconds": 353,
    "title": "The HU - Yuve Yuve Yu (Official Music Video)",
  },
  Object {
    "id": "6iaxDxHUWP8",
    "seconds": 266,
    "title": "Rammstein - Mein Land (Official Video)",
  },
  Object {
    "id": "V9AbeALNVkk",
    "seconds": 392,
    "title": "Twisted Sister - We're Not Gonna Take it (Extended Version) (Official Music Video)",
  },
  Object {
    "id": "7TBYwpQtl5M",
    "seconds": 385,
    "title": "Rammstein - Engel (Hurricane Festival 2016) PROSHOT HD [GER/ENG/RU/ES/FR]",
  },
  Object {
    "id": "hiOjK992bPU",
    "seconds": 243,
    "title": "LINDEMANN - Steh auf (Official Video)",
  },
  Object {
    "id": "WXv31OmnKqQ",
    "seconds": 307,
    "title": "Rammstein - Mein Herz Brennt (Official Video)",
  },
  Object {
    "id": "_lK4cX5xGiQ",
    "seconds": 293,
    "title": "Tenacious D - Tribute (Video)",
  },
  Object {
    "id": "QLGG_GD3iPs",
    "seconds": 257,
    "title": "RAMMSTEIN - Moskau - Москва (перевод)",
  },
  Object {
    "id": "m80G8bomcS0",
    "seconds": 734,
    "title": "Rammstein - Deutschland (live at Milton Keynes, UK 06/07/2019)",
  },
  Object {
    "id": "LIPc1cfS-oQ",
    "seconds": 342,
    "title": "Rammstein - Ohne Dich (Official Video)",
  },
  Object {
    "id": "W3q8Od5qJio",
    "seconds": 236,
    "title": "Rammstein - Du Hast (Official Video)",
  },
  Object {
    "id": "Dji_km6UJvA",
    "seconds": 446,
    "title": "LINDEMANN - Frau & Mann (Official Video)",
  },
  Object {
    "id": "GukNjYQZW8s",
    "seconds": 272,
    "title": "Rammstein - Haifisch (Official Video)",
  },
  Object {
    "id": "uMK0prafzw0",
    "seconds": 293,
    "title": "DIE ANTWOORD - UGLY BOY",
  },
  Object {
    "id": "VJboSby7nW0",
    "seconds": 297,
    "title": "Russian Rammstein - Du Riechst So Gut",
  },
  Object {
    "id": "10JDA8SvwX8",
    "seconds": 235,
    "title": "Rammstein - Stripped (Official Video)",
  },
  Object {
    "id": "xkEju-deJsE",
    "seconds": 202,
    "title": "LINDEMANN - Ach so gern (One Shot Video)",
  },
  Object {
    "id": "EOnSh3QlpbQ",
    "seconds": 246,
    "title": "Rammstein - Ich Will (Official Video)",
  },
  Object {
    "id": "CSvFpBOe8eY",
    "seconds": 207,
    "title": "System Of A Down - Chop Suey! (Official Video)",
  },
  Object {
    "id": "U7IUA8W8xuM",
    "seconds": 331,
    "title": "RAGE AGAINST THE MACHINE - KILLING IN THE NAME - HARDCORE WORLDWIDE (OFFICIAL VERSION HCWW)",
  },
  Object {
    "id": "gNdnVVHfseA",
    "seconds": 228,
    "title": "Rammstein - Mutter (Official Video)",
  },
  Object {
    "id": "uT3SBzmDxGk",
    "seconds": 299,
    "title": "2CELLOS - Thunderstruck [OFFICIAL VIDEO]",
  },
  Object {
    "id": "B9FzVhw8_bY",
    "seconds": 250,
    "title": "The Dead South - In Hell I'll Be In Good Company [Official Music Video]",
  },
  Object {
    "id": "jM8dCGIm6yc",
    "seconds": 397,
    "title": "The HU - Wolf Totem (Official Music Video)",
  },
  Object {
    "id": "cegdR0GiJl4",
    "seconds": 313,
    "title": "Die Antwoord - Enter The Ninja (Explicit Version) (Official Video)",
  },
  Object {
    "id": "v2AC41dglnM",
    "seconds": 293,
    "title": "AC/DC - Thunderstruck (Official Video)",
  },
  Object {
    "id": "IxuEtL7gxoM",
    "seconds": 238,
    "title": "Rammstein - Ich Tu Dir Weh (Official Video)",
  },
  Object {
    "id": "lzeNBRbWXpI",
    "seconds": 310,
    "title": "SABATON - Uprising (Official Music Video)",
  },
  Object {
    "id": "NT714sj011w",
    "seconds": 257,
    "title": "Rammstein - Moskau (Official Audio)",
  },
  Object {
    "id": "Ph-CA_tu5KA",
    "seconds": 217,
    "title": "Rammstein - Links 2 3 4 (Official Video)",
  },
  Object {
    "id": "6fVE8kSM43I",
    "seconds": 214,
    "title": "Slipknot - Duality [OFFICIAL VIDEO]",
  },
  Object {
    "id": "iywaBOMvYLI",
    "seconds": 224,
    "title": "System Of A Down - Toxicity (Official Video)",
  },
  Object {
    "id": "i0VtTMs6Mz4",
    "seconds": 321,
    "title": "Great Big White World - Marilyn Manson",
  },
  Object {
    "id": "cTCXDdHUFE8",
    "seconds": 18,
    "title": "The Passion for Music – Short",
  },
  Object {
    "id": "_S85d4MX7-I",
    "seconds": 181,
    "title": "Worldburn - Sense of Security",
  },
  Object {
    "id": "SDH-Nywqka0",
    "seconds": 224,
    "title": "Worldburn - Shortcut",
  },
  Object {
    "id": "nYIFe8NuIfQ",
    "seconds": 292,
    "title": "Worldburn - Oblivious",
  },
  Object {
    "id": "h0hnpLj6yn0",
    "seconds": 244,
    "title": "Worldburn - Era of Consumption",
  },
  Object {
    "id": "WSiDCsEKMFw",
    "seconds": 227,
    "title": "Worldburn - Shortcut",
  },
  Object {
    "id": "ZbV0eR9lNuw",
    "seconds": 251,
    "title": "Worldburn - Era of Consumption",
  },
  Object {
    "id": "ZUVyRCrmRSU",
    "seconds": 254,
    "title": "Worldburn - A.I.",
  },
  Object {
    "id": "WKX07TdqhWs",
    "seconds": 376,
    "title": "Worldburn - Gluttony",
  },
  Object {
    "id": "-hdn4980H0g",
    "seconds": 462,
    "title": "Worldburn - Process",
  },
  Object {
    "id": "ZzUMrfu1rPY",
    "seconds": 173,
    "title": "Worldburn - Velkavaltio",
  },
  Object {
    "id": "8PLjvuJgD1Y",
    "seconds": 258,
    "title": "Worldburn - Refuse to Suffer",
  },
  Object {
    "id": "1qK4Igowpzg",
    "seconds": 174,
    "title": "Worldburn - Babylonian kahleet",
  },
  Object {
    "id": "GpQRE-hLqQs",
    "seconds": 43,
    "title": "Worldburn - Steroid Penis",
  },
  Object {
    "id": "F2fnsrGCTPk",
    "seconds": 293,
    "title": "Worldburn - Nuclear Flatulence",
  },
  Object {
    "id": "snz37PD6Vms",
    "seconds": 227,
    "title": "Worldburn - Rise",
  },
  Object {
    "id": "EEgUKii2ycM",
    "seconds": 243,
    "title": "Worldburn - Crush the Faithless",
  },
]
`;
